benchmarks/receive/manifests/thanos-receive-controller-deployment.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/component: kubernetes-controller
+    app.kubernetes.io/instance: thanos-receive-controller
+    app.kubernetes.io/name: thanos-receive-controller
+  name: thanos-receive-controller
+  namespace: thanos
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: kubernetes-controller
+      app.kubernetes.io/instance: thanos-receive-controller
+      app.kubernetes.io/name: thanos-receive-controller
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: kubernetes-controller
+        app.kubernetes.io/instance: thanos-receive-controller
+        app.kubernetes.io/name: thanos-receive-controller
+    spec:
+      containers:
+      - args:
+        - --configmap-name=thanos-receive-controller-tenants
+        - --configmap-generated-name=thanos-receive-controller-tenants-generated
+        - --file-name=hashrings.json
+        - --namespace=$(NAMESPACE)
+        env:
+        - name: NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        image: quay.io/observatorium/thanos-receive-controller:master-2020-06-17-a9d9169
```

Comment:
Does this include the latest improvements of yours?

Reply:
No, see description. :wink: 

> The current manifests use Yash's PR as image and the thanos-receive-controller with its latest changes was run on the outside. We should probably clean this up once those merge. :)